versions.yaml

```diff
@@ -14,11 +14,11 @@
 
 module-sets:
   stable:
-    version: v1.0.0-rc10
+    version: v1.0.0-rc9
     modules:
       - go.opentelemetry.io/collector/pdata
   beta:
-    version: v0.76.0
+    version: v0.75.0
```

Comment:
I don't know what is appropriate to do with these prior to cutting another release.